flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/AbstractMetricGroup.java

```diff
@@ -85,9 +86,12 @@
 	 *  For example ["host-7", "taskmanager-2", "window_word_count", "my-mapper" ]. */
 	private final String[] scopeComponents;
 
-	/** The metrics scope represented by this group, as a concatenated string, lazily computed.
+	/** Array the metrics scope represented by this group, as a concatenated string, lazily computed.
```

Comment:
"Array containing the metrics scope represented by this group for each reporter, ..."